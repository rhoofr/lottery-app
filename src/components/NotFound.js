import React from 'react';
import { Link } from 'react-router-dom';
import Page from '../shared/containers/Page';

function NotFound() {
  return (
    <Page title='Page not found'>
      <div className='text-center'>
        <h2>Page cannot be found.</h2>
        <p className='lead text-muted'>
          You can always visit the <Link to='/'>home page</Link> to get a fresh
          start.
        </p>
      </div>
    </Page>
  );
}

export default NotFound;
