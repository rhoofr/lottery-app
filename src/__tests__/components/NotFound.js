import React from 'react';
import { shallow } from 'enzyme';
import NotFound from '../../components/NotFound';

describe('--- NotFound snapshot ---', () => {
  test('should render NotFound Page correctly', () => {
    const wrapper = shallow(<NotFound />);
    expect(wrapper).toMatchSnapshot();
  });
});
