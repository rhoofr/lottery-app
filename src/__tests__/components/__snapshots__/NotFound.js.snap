// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`--- NotFound snapshot --- should render NotFound Page correctly 1`] = `
<Page
  title="Page not found"
>
  <div
    className="text-center"
  >
    <h2>
      Page cannot be found.
    </h2>
    <p
      className="lead text-muted"
    >
      You can always visit the 
      <Link
        to="/"
      >
        home page
      </Link>
       to get a fresh start.
    </p>
  </div>
</Page>
`;
